src/phone_formatter.js

```diff
@@ -241,4 +329,831 @@ class PhoneFormatter extends DelimitedTextFormatter {
   }
 }
 
+const COUNTRY_CODES = [
+  {
```

Comment:
I agree, but I think the simplification of including it here is important.

I think that the right fix to this is to break up field-kit into core and sub formatters so you only need to include what you want/need. I have this pretty close to done (from over 5 months ago) but I need to merge down a bunch of these PRs before a big overhaul is feasible.